opentreemap/opentreemap/celery.py

```diff
@@ -16,3 +19,12 @@
 # pickle the object when using Windows.
 app.config_from_object('django.conf:settings')
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@task_failure.connect
+def handle_task_failure(**kw):
+    rollbar_settings = getattr(settings, 'ROLLBAR', {})
+    access_token = rollbar_settings.get('access_token')
+    if access_token:
+        rollbar.init(access_token, rollbar_settings['environment'])
```

Comment:
Related to https://github.com/rollbar/pyrollbar/blob/master/rollbar/__init__.py#L300. Rollbar is keeping track of calls to init, so we should not try to rig up a similar thing in our code.

I vote yes to moving the `init()` call out to the `celery.py` module so it is called once on load.